src/coreclr/vm/eventing/eventpipe/ep-rt-coreclr.h

```diff
@@ -1691,6 +1691,15 @@ ep_rt_config_value_get_output_streaming (void)
 	return CLRConfig::GetConfigValue (CLRConfig::INTERNAL_EventPipeOutputStreaming) != 0;
 }
 
+static
+inline
+bool
+ep_rt_config_value_get_disable_stacks (void)
```

Comment:
Nit: I'd suggest we name the APIs and fields in positive rather than negative terms so we can avoid all the logic becoming double negatives. Ie: `if(ep_session_get_stacks_enabled()) { // do stack stuff }`